### DISCUSSION TITLE
Nocache tags on 404 pages

### DISCUSSION THREAD

**Discussion by byron-roots** [statamic/cms#10762] [Troubleshooting] [answered]
We've been having some issues with our redis setup and it constantly reaching its memory limit. We've recently migrated a few sites over to the nocache DB driver.

Having a look at the table it seems to be excessively larger. One site has roughly 220,000 rows (863MB) whilst the next largest table is 27MB. I think this is due to nocache tags on our 404 pages, and then crawlers/bots landing on pages that don't exist.

I've removed the no cache tag regions from our 404 pages to try and prevent this, but is there a better way of handling this while having nocache tags? I also image pages with query strings are used for filtering content will be contributing to this as well where we can't really remove nocache tags.

**Comment by jasonvarga** [comment, marked as answer]
Yes, you can enable the shared errors option. All 404s would be considered the same page, rather than a separate cached page for every missing page.

See the PR: https://github.com/statamic/cms/pull/10294

In `config/statamic/static_caching.php` set `'share_errors' => true`

This has not been documented yet though. https://github.com/statamic/docs/issues/1376

**Reply by byron-roots** [reply]
Perfect, thanks!
